.github/workflows/test-images.yaml

```diff
@@ -0,0 +1,71 @@
+name: Create Test Images
+on:
+  push:
+    paths:
+      - 'test/e2e/fixtures/Dockerfile.*'
+  workflow_dispatch:
+
+
+jobs:
+  list-dockerfiles:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'workflow_dispatch' }}
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: set-matrix
+        run: echo "::set-output name=matrix::$(ls test/e2e/fixtures/Dockerfile.* | jq -R -s -c 'split("\n")[:-1]')"
```

Comment:
Just to check - is `jq` installed on `ubuntu-latest`?

Reply:
Any chance you can kick this off manually in your fork to check this? (though I appreciate that you might need to update the registry for that.

Reply:
Also - it looks like `set-output` is deprecated now: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
Thanks for the fixes!